compiler/rustc_passes/src/dead.rs

```diff
@@ -1136,7 +1167,11 @@ impl<'tcx> DeadVisitor<'tcx> {
 }
 
 fn check_mod_deathness(tcx: TyCtxt<'_>, module: LocalModDefId) {
-    let (live_symbols, ignored_derived_traits) = tcx.live_symbols_and_ignored_derived_traits(());
+    let live_symbols_result = tcx.live_symbols_and_ignored_derived_traits(());
+    if live_symbols_result.is_err() {
+        return;
+    }
+    let (live_symbols, ignored_derived_traits) = live_symbols_result.as_ref().unwrap();
```

Comment:
Use `let Ok(..) else {return} ` instead of checking a value and then unwrapping it